services/explorer-ui/src/api/block.ts

```diff
@@ -15,28 +15,28 @@ export const BlockAPI = {
   },
   getBlockByHeight: async (height: string): Promise<ChicmozL2Block> => {
     const response = await client.get(
-      `${aztecExplorer.getL2BlockByHeight}${height}`,
+      `${aztecExplorer.getL2BlockByHeight}${height}`
     );
     return validateResponse(chicmozL2BlockSchema, response.data);
   },
   getBlocksByHeightRange: async (
-    start: number,
-    end: number,
+    start?: number,
```

Comment:
why do we make these optional?

Reply:
Because I wanted you to be able to query without params and get the last 30 blocks. (Easier for us in the UI)